### PR TITLE
Firestore populated grids

### PR DESCRIPTION
**Pull request by Rhaypapenfuzz**

### CODE REVIEW

File: src/Profile.js [hunk 1]
```diff
@@ -50,62 +50,19 @@ export default function CreatedGrid() {
...
-//         </Card>
-//       );
-//     })}
-//   </div>
-// );
-// }
+    return (
+      <div className="board">
+        <img src={profilpic} alt="Girl in a jacket" width="150" height="150" />
+        {grid_list.map((grid, i) => {
```
Comment (kricheso) [inline src/Profile.js]: What is i?

Comment (kricheso) [reply src/Profile.js]: Not important, but I noticed it was never used. Resolving

**Review by kricheso — APPROVED** [review]
Good.

File: src/Profile.js (hunk 1 above)
Comment (Rhaypapenfuzz) [reply src/Profile.js]: that was what swarnim sent into the hangout when he couldn't pull and push. I just used what is with master now

**Pull request merged**